apps/patient/src/views/Pharmacy.tsx

```diff
@@ -132,6 +132,9 @@ export const Pharmacy = () => {
   const hasTopRankedCostco = topRankedPharmacies.some((p) => p.name === 'Costco Pharmacy');
   const enableMailOrder =
     !isDemo &&
+    // If we're loading we shouldn't show pharmacies of mail order (we don't even know if we want to show until
+    // we see if there are top ranked costcos)
+    !loadingPharmacies &&
```

Comment:
This is only semi-related, ui fix. Without this two weird behaviors arise:
- You can see mail order before local pharmacies load
- With costco, it was showing mail order until pharmacies load, after which it would hide the mail order

The latter couldve also been fixed by checking if toprankedwasenabled

Reply:
Coming back, doing the latter instead to avoid other design considerations